test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -137,15 +181,51 @@ string FixUrl(string url)
             }
         }
 
-        private async Task RegisterIPEndPoint_Success(IPEndPoint endPoint, Func<IPEndPoint, string> testUrl)
+        private Task RegisterAddresses_Success(string addressInput, string testUrl, bool appendPortToTestUrl)
+            => RegisterAddresses_Success(addressInput, new[] { testUrl }, appendPortToTestUrl);
+
+        private async Task RegisterAddresses_StaticPort_Success(string addressInput, string[] testUrls)
+        {
+            var retryCount = 0;
+            var errors = new List<Exception>();
+
+            while (retryCount < MaxRetries)
```

Comment:
I thought of a delay too, but @halter73 pointed out that we're getting a new port each time, so why delay?